cap-template.md

```diff
@@ -30,19 +59,41 @@ A short (~200 word) description of the technical issue being addressed.
 ## Specification
 The technical specification should describe the syntax and semantics of any new feature.
 
+### xdr changes
```

Comment:
Not sure if this was intentional or a typo. Is there a reason we don't capitalize XDR which is an abbreviation?
```suggestion
### XDR changes
```

Reply:
I noticed the same thing. I also think it should be XDR.